docs/topics/circuit-breaker-mission-access-hystrix-dashboard-spring-boot.adoc

```diff
@@ -0,0 +1,57 @@
+= Using the Hystrix Dashboard to Monitor the Circuit Breaker Application
+
+Hystrix Dashboard lets you easily monitor the health of your services in real time by aggregating Hystrix metrics data from an event stream and displaying them on one screen. For more detail, see the Hystrix Dashboard link:https://github.com/Netflix/Hystrix/wiki/Dashboard[wiki page].
+
+NOTE: You must have the Circuit Breaker booster application deployed before proceeding with the steps below.
+
+//  Openshift first. Do we reference Minishift to? Add a note or a section?
+// Replace with proper OSO URLs
+. Log in to your {OpenShiftOnline} instance.
```

Comment:
Correct. IIRC we should focus on OpenShift Online online rather than Minishift in our docs. I'll change the URLs to match the OSO format and update the PR